sdk/communication/azure-communication-chat/azure/communication/chat/_version.py

```diff
@@ -4,6 +4,6 @@
 # license information.
 # --------------------------------------------------------------------------
 
-VERSION = "1.2.0"
+VERSION = "2.0.0-beta.1"
```

Comment:
pls follow the python versioning [policy](https://azure.github.io/azure-sdk/policies_releases.html#python), I guess it will also solve the pipeline issue

Reply:
@AikoBB please explain in more detail how this should be changed. I see in the policy that "Beta releases use a prerelease label like -beta.X" and as he have a major version upgrade, I've also changed the major version here.

Reply:
```suggestion
VERSION = "2.0.0b1"
```